project-addons/purchase_picking/stock.py

```diff
@@ -20,14 +20,29 @@
 ##############################################################################
 
 from openerp import models, fields, api, _, exceptions
+import datetime
 
 
 class StockContainer(models.Model):
 
     _name = "stock.container"
 
+    @api.multi
```

Comment:
El campo date_expected estaba pensado para que si se escribiría actualizara el min_date de todos los movimientos, esta funcionalidad se pierde ¿no? Si la queréis perder habría que borrar la lógica del método write, en este mismo fichero.
Sino, creo que este código encaja más como el default del campo no como un function